blog/_posts/2017-12-04-multilingual-chatrooms.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: blog-detail
+post-type: blog
+by: Jon Pretty
+title: "Introducing Non-English Chatrooms for Scala Users"
+---
+
+I attend a lot of conferences around the world, and usually all the talks and
+most of the communication among attendees happens in English. It has become (if
+you'll excuse the irony) the *lingua franca* of the Scala community, and the
+wider software industry, even though speakers of English as a first language
+are a minority.
+
+There are, however, many users and potential users of Scala who either do not
+speak English, or do not have the confidence to communicate in public forums in
+English, and the expectation that English be a prerequisite to becoming an
+effective Scala programmer disproportionately handicaps many minority groups
+from countries outside the USA and the rest of the English-speaking world.
+
+So I was surprised to discover that for most non-English languages, there were
+no online communities where people could ask questions in their native tongue.
+And so, it took very little effort to create several channels on
+[Gitter](https://gitter.im/) dedicated to talking about Scala in a variety of
+non-English languages:
+
+- [scala/fr](https://gitter.im/scala/fr) — Français
+- [scala/de](https://gitter.im/scala/de) — Deutsch
+- [scala/it](https://gitter.im/scala/it) — Italiano
+- [scala/es](https://gitter.im/scala/es) — Español
+- [scala/pl](https://gitter.im/scala/pl) — Polski
+
+So far, only this handful of channels exists, and we encourage any speakers of
+these languages to join one of these groups, but after just a few hours online,
+some of these channels are attracting lots of activity. Should demand exist for
+other language groups with a significant number of speakers, we would be happy
+to facilitate these by creating new channels as necessary. Please get in touch
+with me, [Jon Pretty](https://twitter.com/propensive/), if you would like to
+suggest a new channel.
+
+We do not anticipate English losing its primacy as the common language of the
```

Comment:
I would just mention that sometimes unconsciously people do not contribute to chats because they don't feel part of the group. One of the reasons is cultural, but another reason is emotional too -- being able to speak in your mother tongue encourages you to be more participative and eager to help (at least, in my experience).

Reply:
Sure, sounds reasonable. I'll amend it.